test/cpp/end2end/test_service_impl.cc

```diff
@@ -464,44 +526,247 @@ Status TestServiceImpl::BidiStream(
   return Status::OK;
 }
 
-namespace {
-int GetIntValueFromMetadataHelper(
-    const char* key,
-    const std::multimap<grpc::string_ref, grpc::string_ref>& metadata,
-    int default_value) {
-  if (metadata.find(key) != metadata.end()) {
-    std::istringstream iss(ToString(metadata.find(key)->second));
-    iss >> default_value;
-    gpr_log(GPR_INFO, "%s : %d", key, default_value);
-  }
+void CallbackTestServiceImpl::RequestStream(
```

Comment:
Aha. Took me a bit to realize RequestStream is the name of the method, and this isn't an async-style server-side request to provision an incoming "Stream" RPC.

Reply:
Right, which leads to hideousness like RequestRequestStream in the CQ-based api.